### ISSUE TITLE
Link missing text alternative

### ISSUE THREAD

**Issue by alomarramaa** [samvera/hyku#3090]
https://vpat.hykudemo.org

Version 7.1

### Descriptive Summary

Link missing text alternative (for all of these links under Recent Works)
If an image link is missing an accessible name, screen reader users will not know what the link is supposed to do. In this case, many screen readers will read the URL of the link instead. 

### Expected Behavior
Ensure that <a> elements, <area> elements, and elements with a role=link attribute, have an accessible name.

### Steps to Reproduce Issue
1. Go to vpat.hykudemo.org
2. Use SiteImprove extension to scan the page

### Occurrences of Issue

This issue is found on the home page of the Culture Repository theme.

<img width="561" height="346" alt="Image" src="https://github.com/user-attachments/assets/dc0dcd53-8842-472d-b9f7-d3a8108e943d" />
